WordPressKitTests/BlogServiceRemoteRESTTests+Jetpack.swift

```diff
@@ -0,0 +1,197 @@
+import Foundation
+import XCTest
+@testable import WordPressKit
+
+class BlogServiceRemoteRESTTests_Jetpack: RemoteTestCase, RESTTestable {
+    let siteId = 12345
+    let url = "http://www.wordpress.com"
+    let encodedURL = "http%3A%2F%2Fwww.wordpress.com"
+    let username = "username"
+    let password = "qwertyuiop"
+    
+    let jetpackRemoteSuccessMockFilename = "blog-service-jetpack-remote-success.json"
+    let jetpackRemoteFailureMockFilename = "blog-service-jetpack-remote-failure.json"
+
+    let jetpackRemoteErrorUnknownMockFilename = "blog-service-jetpack-remote-error-unknown.json"
+    let jetpackRemoteErrorInvalidCredentialsMockFilename = "blog-service-jetpack-remote-error-invalid-credentials.json"
+    let jetpackRemoteErrorForbiddenMockFilename = "blog-service-jetpack-remote-error-forbidden.json"
+    let jetpackRemoteErrorInstallFailureMockFilename = "blog-service-jetpack-remote-error-install-failure.json"
+    let jetpackRemoteErrorInstallResponseMockFilename = "blog-service-jetpack-remote-error-install-response.json"
+    let jetpackRemoteErrorLoginFailureMockFilename = "blog-service-jetpack-remote-error-login-failure.json"
+    let jetpackRemoteErrorSiteIsJetpackMockFilename = "blog-service-jetpack-remote-error-site-is-jetpack.json"
+    let jetpackRemoteErrorActivationInstallMockFilename = "blog-service-jetpack-remote-error-activation-install.json"
+    let jetpackRemoteErrorActivationResponseMockFilename = "blog-service-jetpack-remote-error-activation-response.json"
+    let jetpackRemoteErrorActivationFailureMockFilename = "blog-service-jetpack-remote-error-activation-failure.json"
+    
+    var endpoint: String { return "jetpack-install/\(encodedURL)/?locale=en_US" }
```

Comment:
`?locale=en_US`
Just a note to remove locale here as well if its indeed unnecessary.